packages/extensions/core/src/lib/plugins/merger.ts

```diff
@@ -135,7 +135,7 @@ export class MultiAPIMerger extends Transformer<any, oai.Model> {
         case "info":
           if (!this.isSecondaryFile) {
             const info = this.getOrCreateObject(target, "info", pointer);
```

Comment:
maybe you can type info as AnyObject instead of the any cast below?

```ts
const info: AnyObject = this.getOrCreateObject(target, "info", pointer);

```

Reply:
actually was able to update the type of getOrCreateObject to be accurate